tests/VolleyManagement.UnitTests/Services/GameService/GameServiceTests.cs

```diff
@@ -1331,6 +1331,37 @@ public void Create_FifthSetValidScore_GameCreated()
             VerifyCreateGame(newGame, Times.Once());
         }
 
+        [TestMethod]
+        public void Create_SameTeamInTheRound_ExceptionThrown()
+        {
+            MockTournamentSchemeTwo();
```

Comment:
Why scheme 2?

Reply:
Because I am testing the case when the scheme is not PlayOff :) 
Must I name this TestMethod as Create_SameTeamInTheRoundSchemeTwo_ExceptionThrown and create another one for scheme one?